tests/drivers/si1133/main.c

```diff
@@ -21,6 +21,7 @@
 #include <stdio.h>
 #include <inttypes.h>
```

Comment:
```suggestion
#include <stdio.h>
#include <stdbool.h>
#include <inttypes.h>
```

I'm not sure why this error is triggered by adding `kernel_defines.h` tbh 🤔